test/src/index.test.js

```diff
@@ -281,34 +312,50 @@ describe('Anonymizer', () => {
         });
       });
 
-      it('should not trim obfuscated values that have different obfuscation techniques', () => {
-        const replacement = (key, value) => {
-          if (key === 'biz') {
-            return value;
-          }
-
-          if (value === 'bux') {
-            return '--HIDDEN--';
-          }
-
-          return '--REDACTED--';
-        };
-        const anonymize = anonymizer({ whitelist: ['foo'] }, { replacement, trim: true });
+      it('should trim obfuscated fields without adding their paths to a `__redacted__` list', () => {
+        const anonymize = anonymizer({ whitelist: ['foo'] }, { strategy: TRIM });
 
         expect(
           anonymize({
             biz: 'baz',
-            buz: 'bux',
-            foo: 'bar',
-            qux: 'quux'
+            buz: { bux: { qux: 'quux' } },
+            foo: 'bar'
           })
         ).toEqual({
-          __redacted__: ['qux'],
-          biz: 'baz',
-          buz: '--HIDDEN--',
           foo: 'bar'
         });
       });
+
+      [TRIM, TRIM_AND_LIST].forEach(strategy => {
```

Comment:
No need, list is small, split into two `it`.

Reply:
Done